model/otel/metrics.yaml

```diff
@@ -0,0 +1,105 @@
+groups:
+  - id: metric.otel.sdk.span.created_count
+    type: metric
+    metric_name: otel.sdk.span.created_count
+    stability: development
+    brief: "The number of spans which have been created"
+    instrument: counter
+    unit: "{span}"
+    attributes:
+      - ref: otel.span.is_sampled
+
+  - id: metric.otel.sdk.span.live_count
+    type: metric
+    metric_name: otel.sdk.span.live_count
+    stability: development
+    brief: "The number of created spans for which the end operation has not been called yet"
+    instrument: updowncounter
+    unit: "{span}"
+    attributes:
+      - ref: otel.span.is_sampled
+
+  - id: metric.otel.sdk.span.ended_count
+    type: metric
+    metric_name: otel.sdk.span.ended_count
+    stability: development
+    brief: "The number of created spans for which the end operation was called"
+    instrument: counter
+    unit: "{span}"
+    attributes:
```

Comment:
Feedback from implementing this metric: These "sdk" metrics are really metrics about the behavior of an instance SDK tracerprovider.  One thing i've encountered is that I need an "ID" or "name" to differentiate instances of the SDK tracerprovider similar to the batch span processor below.

We should consider adding the otel.sdk.component.name (or an otel.sdk.component.id attribute) to this and the created/live count metrics as well.  I'm not sure otel.sdk.component.type makes sense since it will always be the TracerProvider.